README.md

```diff
@@ -1,129 +1,203 @@
+    1. [Branch Convention](#branch-convention)
+    1. [Review Process](#review-process)
+1. [Documentation](#documentation)
+1. [License](#license)
+1. [Authors](#authors)
+1. [Acknowledgments](#acknowledgements)
+
+## About the Project
+
+This repository contains all code related to [Longhorn Racing Solar](https://www.longhornracing.org/solar-vehicle-team)'s Controls System, affiliated with the **University of Texas at Austin**. The Controls System is responsible for driver interactions, motor control, and lighting systems control. The system is built on [custom hardware](https://github.com/lhr-solar/Controls-LeaderPCB) and an ARM Cortex-M4 Processor (STM32F413).
+
+For more information, see:
+- [API documentation](https://controls-docs.readthedocs.io/en/latest/)
+- [LHR Solar Wiki](https://wikis.utexas.edu/display/LHRSOLAR/Controls+Documentation)
+
+**[Back to top](#table-of-contents)**
+
+## Project Status
+
+The Controls System is currently on release **M2.0** for the 2022-24 build cycle to compete in [FSGP 2024](https://www.americansolarchallenge.org/the-competition/2024-american-solar-challenge/). See our [Versioning](#versioning).
+
+LHR Solar uses ClickUp as our project management tool. To access the [our ClickUp board](https://app.clickup.com/9011033583/v/f/90110413188), make sure a system lead has added you as a member.
+
+We also use GitHub Projects to track software releases. See our [active projects](https://github.com/lhr-solar/Controls/projects?query=is%3Aopen).
+
+**[Back to top](#table-of-contents)**
+
+## Getting Started
+
+Please review [Setting Up an Embedded Development Environment](https://wikis.utexas.edu/display/LHRSOLAR/Setting+up+an+Embedded+Development+Environment) on our wiki. This should get you up and running with some sort of Linux machine.
+
+### Workspace Setup
+
+1. Clone this repository via [SSH](https://docs.github.com/en/authentication/connecting-to-github-with-ssh): ```git clone --recurse-submodules git@github.com:lhr-solar/Controls.git```. Make sure to include the --recurse-submodules flag, or run ```git submodule update --init --recursive``` to pull all necessary submodules.
+
+1. In VSCode, remote into your Linux machine if needed. Click **File** &#8594; **Open Workspace from File** and select the workspace located in ```.vscode/LHR.code-workspace```. The VSCode Workspace has many integrations that developers may find useful while debugging or writing code, including but not limited to integrations with [Renode Simulator](#renode-simulator) and the openocd on-chip debugger.
+
+1. Run ```Embedded-Sharepoint/Scripts/install.sh``` to selectively install your packages. For Controls development, you will need the following packages:
+    * ARM Toolchain
+    * build-essential
+    * gdb-multiarch
+    * openocd
+    * stlink-tools
+    * Renode Simulator
+    * Documentation Tools (Sphinx, Doxygen, and Breathe)
+
+    The script will prompt for installation of each of these packages.
```

Comment:
Need to tell them to clone the embedded sharepoint

Reply:
git submodule update will do that once it's a submodule ig.